auth.env.sample

```diff
@@ -0,0 +1,2 @@
+TEST_USER_UID=uid
```

Comment:
As far as I understand, this is used only for the unit tests. Probably it would be better to create a new user, and then delete it somewhere in the tests? Otherwise, in case of changing the `firebase.json` or removing this user, the tests will fail.

Reply:
Yes it would, I hadn't thought that